docs/source/index.md

```diff
@@ -2,6 +2,38 @@
 
 ## User documentation:
 
+[The JupyterLab Interface](./user/interface.rst)
+
+[JupyterLab URLs](./user/urls.rst)
+
+[Working with Files](./user/files.rst)
```

Comment:
Please, combine "Working with Files" with "File management" inside of "Working with Files" and delete "File management"